### PR TITLE
add readmo cask

### PR DESCRIPTION
**Pull request by FinalDes**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please add a livecheck and a zap stanza.

**Comment by FinalDes** [comment]
I need some help to add livecheck and zap stanza

File: Casks/readmo.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 "997da8763676b40846876679b1d9541df93ee84e91ecfd5c024659ff8608e2b2"
+
+  url "https://cdn.readmoo.com/download/apps/desktop/osx64/el/Readmoo_Desktop-#{version}.dmg"
+  name "readmo"
+  desc "Readmoo讀墨電子書 ─ 台灣最大繁體中文EPUB電子書服務"
+  homepage "https://readmoo.com/"
+
+  livecheck do
+    url "https://readmoo.com/download/osx"
+  end
```
Comment (p-linnane) [inline Casks/readmo.rb:12]:
```suggestion
  livecheck do
    url "https://readmoo.com/download/osx"
    strategy :header_match
  end
```

File: Casks/readmo.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  app "Readmoo看書.app"
+
+  zap trash: [
+    "/Users/alex/Library/Logs/Readmoo看書",
+    "/Users/alex/Library/Application\ Support/Readmoo看書"
```
Comment (p-linnane) [inline Casks/readmo.rb:18]:
```suggestion
    "~/Library/Logs/Readmoo看書",
    "~/Library/Application Support/Readmoo看書",
```

**Comment by p-linnane** [comment]
I've added suggestions to fix the `livecheck` and `zap` stanzas. Please add a description for this cask.

**Comment by miccal** [comment]
Thank you @FinalDes.

**Review by miccal — APPROVED** [review]

**Pull request merged**